packages/utilities/psammead-styles/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.2.1 | [PR#3020](https://github.com/bbc/psammead/pull/3020) Add colour C_LEARNINGENGLISH and set font families to Latin Reith for Learning English |
```

Comment:
This should be a minor version update since we're adding new exports. 
```suggestion
| 4.3.0 | [PR#3020](https://github.com/bbc/psammead/pull/3020) Add colour C_LEARNINGENGLISH and set font families to Latin Reith for Learning English |
```